6_visualize/log/6_gif_tasks.ind

```diff
@@ -0,0 +1,192 @@
+6_visualize/tmp/gif_frame_a_20170829_23.png: b2e0ca876b770f54bd0b70dd9eebd090
+6_visualize/tmp/gif_frame_a_20170830_00.png: cb4e3df3b4fa0fd2d6fcf594f53b4cb7
+6_visualize/tmp/gif_frame_a_20170830_01.png: 224e43291211930e60fbd924b419a21e
+6_visualize/tmp/gif_frame_a_20170830_02.png: abaa7e176cfd7142ec8bd4908a3f0b74
+6_visualize/tmp/gif_frame_a_20170830_03.png: 0c73ae6dc953ec0eaa5ed31ba0611758
+6_visualize/tmp/gif_frame_a_20170830_04.png: d8989daefca3ef3cce6a06b23c998d14
+6_visualize/tmp/gif_frame_a_20170830_05.png: 56ca7a43c57957dc5754de2fba6db03a
+6_visualize/tmp/gif_frame_a_20170830_06.png: 9cf14fd5bcbcedee68228318a93fea16
+6_visualize/tmp/gif_frame_a_20170830_07.png: dc8a6e91514788c1aa46006e7444345b
+6_visualize/tmp/gif_frame_a_20170830_08.png: b4a7b0c3635b398dfdd0bc9892e0e14f
+6_visualize/tmp/gif_frame_a_20170830_09.png: 534adabfde0ee247dd4592a1de72a13d
+6_visualize/tmp/gif_frame_a_20170830_10.png: facdb84cab622fc3ac28620c2893f631
+6_visualize/tmp/gif_frame_a_20170830_11.png: deda0a103b5792701d0eb2770f58a1c2
+6_visualize/tmp/gif_frame_a_20170830_12.png: c6d03f83edde7beda3bc72719b6f3ecc
+6_visualize/tmp/gif_frame_a_20170830_13.png: ddac516bd4bb4e2708fd7f708aafe9a4
+6_visualize/tmp/gif_frame_a_20170830_14.png: f93825f1a428063663c635ea568a0792
+6_visualize/tmp/gif_frame_a_20170830_15.png: e94e480f31b6e5c8fa5069b1ab7e3de3
+6_visualize/tmp/gif_frame_a_20170830_16.png: d50cecf984e8378a796acf799d981f00
+6_visualize/tmp/gif_frame_a_20170830_17.png: 9b01049ddde073801606635c3dd9e496
+6_visualize/tmp/gif_frame_a_20170830_18.png: fbbfe0a2e1c8f4850f67cfbfea7ef64f
+6_visualize/tmp/gif_frame_a_20170830_19.png: 6ef373ae65dcac750d9767b5314c252f
+6_visualize/tmp/gif_frame_a_20170830_20.png: 7352b7a722355fcde57e631704ad0b6c
+6_visualize/tmp/gif_frame_a_20170830_21.png: 7e5dd555f0f95fb3042c03714319d2ab
+6_visualize/tmp/gif_frame_a_20170830_22.png: beb4328f5b63f33061be098867f89121
+6_visualize/tmp/gif_frame_a_20170830_23.png: a9d1c4addeffdce87ffef5d7758ef9ee
+6_visualize/tmp/gif_frame_a_20170831_00.png: 557050c73d74c630a5ac12dc0fccdf2c
+6_visualize/tmp/gif_frame_a_20170831_01.png: f6ac7cd64e0902820f554bc01a1f8895
+6_visualize/tmp/gif_frame_a_20170831_02.png: 3b022323a4d1a4e7b88b4aab45791e27
+6_visualize/tmp/gif_frame_a_20170831_03.png: f3d14d0119a667386a9dfb9e57db6e44
+6_visualize/tmp/gif_frame_a_20170831_04.png: ea6395d4bce3402e391bd1a815d14376
+6_visualize/tmp/gif_frame_a_20170831_05.png: ab491a27b014593a03ec2dfcb50269fe
+6_visualize/tmp/gif_frame_a_20170831_06.png: bf9e345f85d7d0e29c87764671dfa13c
+6_visualize/tmp/gif_frame_a_20170831_07.png: 3025c43ecc3ab7db900aaa82dc2502dc
+6_visualize/tmp/gif_frame_a_20170831_08.png: e2b264600c7c2906555a1150854e25dd
+6_visualize/tmp/gif_frame_a_20170831_09.png: 171bd48cca2d210e28d841807227bcec
+6_visualize/tmp/gif_frame_a_20170831_10.png: 45c249d706248d90bc162688cf8bda2b
+6_visualize/tmp/gif_frame_a_20170831_11.png: 4ab86ba76c434dcafd68eb855214787b
```

Comment:
wheeee! was this fast?

Reply:
Fast enough - seconds per frame, but a lot of frames...

Reply:
total build was 10mins

Reply:
Noted. Could be worse, and I'm still hopeful that this multi-file approach will get us the iteration speed we want even if the total runtime is nontrivial.